File: prog2_pentek/02/01/Book.java


public class Book {

    public static void main(String[] args) {

        konyv konyv1 = new konyv("Harry Potter", "J.K. Rowling", 400); //Ez itt egy objektum, amit a konyv osztalybol hoztunk letre.

        konyv konyv2 = new konyv("Alapitvany", "Asimow", 255);

        konyv1.pages = 500;
        System.out.println("Oldalszam: " + konyv1.pages);
        System.out.println("Cim: " + konyv1.title);
        System.out.println("Cim: " + konyv2.title);
        
    }
    
}

File: prog2_pentek/02/01/book.c
#include <stdio.h>

typedef char * string;

typedef struct {
    string title;
    string author;
    int pages;
}

int main()
{
    Book book1;
    book1.title = "dune";
    book1.author = "Frank Herbert";
    book1.pages = 1;

    printf("%s\n", book1.title);

    return 0;
}